classes/PSElementTrait.php

```diff
@@ -226,6 +226,27 @@ public function sync()
     {
         SRBLogger::addLog('SYNCHRONIZING ' . self::getObjectTypeForMapping() . ' "' . $this->getReference() . '"', SRBLogger::INFO, self::getObjectTypeForMapping(), $this->getDBId());
 
+        // To manage product duplication
+        //TODO generalize to other Elements
```

Comment:
does this TODO has a corresponding open issue ?

Reply:
Now it does :smile: